test/com/facebook/buck/rules/DefaultKnownBuildRuleTypes.java

```diff
@@ -23,13 +23,20 @@
 import com.facebook.buck.util.FakeProcess;
 import com.facebook.buck.util.FakeProcessExecutor;
 import com.facebook.buck.util.ProcessExecutorParams;
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Optional;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
```

Comment:
I think it was flagging a different one, which is fixed now.

This one is used below in `getPaths` to create an `emptyList()`.